tests/Unit/AsyncPFE/async_av_dependent7.cpp

```diff
@@ -85,30 +84,28 @@ bool test1D() {
 
   // now there must be 1 pending async operations for the accelerator_view
   // pfe1 and pfe2 must be completed by now
-  ret &= (hc::accelerator().get_default_view().get_pending_async_ops() == 1);
+  assert(hc::accelerator().get_default_view().get_pending_async_ops() == 1);
 
   // for this test case we deliberately NOT wait on kernels
   // we want to check when array_view instances go to destruction
   // would all dependent kernels be waited or not 
-
-  return ret;
 }
 
 int main() {
   bool ret = true;
 
   hc::accelerator_view av = hc::accelerator().get_default_view();
 
-  ret &= test1D<32, 16>();
-  ret &= (av.get_pending_async_ops() == 0);
-  ret &= test1D<64, 8>();
-  ret &= (av.get_pending_async_ops() == 0);
-  ret &= test1D<128, 32>();
-  ret &= (av.get_pending_async_ops() == 0);
-  ret &= test1D<256, 64>();
-  ret &= (av.get_pending_async_ops() == 0);
-  ret &= test1D<1024, 256>();
-  ret &= (av.get_pending_async_ops() == 0);
+  test1D<32, 16>();
+  assert(av.get_pending_async_ops() == 0);
+  test1D<64, 8>();
+  assert(av.get_pending_async_ops() == 0);
+  test1D<128, 32>();
+  assert(av.get_pending_async_ops() == 0);
+  test1D<256, 64>();
+  assert(av.get_pending_async_ops() == 0);
+  test1D<1024, 256>();
+  assert(av.get_pending_async_ops() == 0);
```

Comment:
Also I only added HCC_OPT_FLUSH define to tests where the HCC_OPT_FLUSH=1 changed the number of expected ops.